### ISSUE TITLE
The loss  remains high and the accuracy is very low

### ISSUE THREAD

**Issue by tangdk** [yangyuke001/FIIQA-PyTorch#11]
Hi,
I started training from scratch without using the pre_trained model **97_160_2.pth**
My training results were poor
![image](https://user-images.githubusercontent.com/18397441/127254916-b41d0e43-863a-42b1-859e-f30ec30570c0.png)

I use FIIQALoss(),and optimizer is AdaBound.
Even if I loaded the pre_trained model, I didn't get better results, why?
Thanks
